{"name": "test_api2[张三-18]", "status": "broken", "statusDetails": {"message": "NameError: name 'YamlUtil' is not defined", "trace": "@pytest.fixture(scope=\"session\", autouse=True)\n    def clear_yaml():\n>       YamlUtil().clear_extract_yaml()\nE       NameError: name 'YamlUtil' is not defined\n\nconftest.py:14: NameError"}, "parameters": [{"name": "name", "value": "'张三'"}, {"name": "age", "value": "18"}], "start": 1669275299103, "stop": 1669275299103, "uuid": "05b52fc0-365e-47f4-8d79-7a5570fd20c1", "historyId": "47393bfab1934db98a729dc4db3ab01c", "testCaseId": "a7b6fe78ffeca118517d048a45bbe491", "fullName": "testcase.test _api.TestApi#test_api2", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test _api"}, {"name": "subSuite", "value": "TestApi"}, {"name": "host", "value": "admin"}, {"name": "thread", "value": "55928-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test _api"}]}